### PR TITLE
CORCI-885 build: Removing SLES-12, JHLI rpm repos.

### PR DESCRIPTION
**Pull request by JohnMalmberg**
Patch an issue with the Leap-15 dockerfile certificates.

Remove the remaining files for SLES-12 which we can no longer support.

Remove the JHLI rpm repositories as ipmctl is now available from
the normal repositories for a distro.

Signed-off-by: John E Malmberg <john.e.malmberg@intel.com>

### CODE REVIEW

**Comment by JohnMalmberg** [comment]
This should not have any changes that would cause test results differences.

File: utils/docker/Dockerfile.leap.15 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2019, Intel Corporation
+# Copyright 2020, Intel Corporation
```
[the remark this answers is not in the document]
Comment (JohnMalmberg) [reply utils/docker/Dockerfile.leap.15]: Fixing.

**Comment by JohnMalmberg** [comment]
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/master/1892/ has the same test failures.

**Comment by JohnMalmberg** [comment]
Ken Cain saw the LlnlMpi4py failure on one of my PRs, and filed DAOS-4368 for that.

File: utils/docker/Dockerfile.centos.7 [hunk 2]
```diff
@@ -67,13 +67,11 @@ RUN yum -y install epel-release; \
...
            python2-avocado python2-avocado-plugins-output-html \
            python2-avocado-plugins-varianter-yaml-to-mux       \
-           yum-plugin-copr python-pathlib                      \
+           python-pathlib                                      \
            ndctl ipmctl e2fsprogs                              \
            python2-clustershell python2-Cython python2-pip     \
            python36-clustershell python36-paramiko             \
            python36-numpy python36-jira python3-pip;           \
-    yum -y copr enable jhli/ipmctl;                            \
-    yum -y copr enable jhli/safeclib;                          \
```
Comment (mjmac) [inline utils/docker/Dockerfile.centos.7]: Where will these packages come from? They don't appear to be available in EPEL.

Comment (JohnMalmberg) [reply utils/docker/Dockerfile.centos.7]: The current ipmctl package is now only available in EPEL.  The safeclib was a dependency of it.  I have not looked to see if it is no longer a dependency or if it has a new name in EPEL.

The copr library is no longer being referenced by the ipmctl home page.

Comment (mjmac) [reply utils/docker/Dockerfile.centos.7]: Oh, never mind. I guess they are in EPEL now. Sorry for the noise.

**Comment by JohnMalmberg** [comment]
Yes, please force land this.

**Pull request merged**